packages/puppeteer-core/src/bidi/core/UserPrompt.ts

```diff
@@ -76,31 +90,35 @@ export class UserPrompt extends EventEmitter<{
       }
       this.#result = parameters;
       this.emit('handled', parameters);
-      this.dispose('User prompt was handled.');
+      this.dispose('User prompt already handled.');
     });
   }
 
   // keep-sorted start block=yes
   get #session() {
     return this.browsingContext.userContext.browser.session;
   }
+  get closed(): boolean {
+    return this.#reason !== undefined;
+  }
   get disposed(): boolean {
-    return Boolean(this.#reason);
+    return this.closed;
+  }
+  get handled(): boolean {
```

Comment:
It feels like in a lot of places we have duplicates getters, happy to merge as is but let's align offline on this.